### ISSUE TITLE
Investigate photo meta data for non-jpeg photos

### ISSUE THREAD

**Issue by charliepnnl** [pnnl/Quality-Install-Tool#46]
The meta data module we are using, `exif-js` works for jpeg photos only.  Is it possible for us to support other photo types also?

This is especially important for new iPhones which, by default, use the ??? format.

**Comment by sudhacheran** [comment]
I have tested live QI app for HEIC format 
1. iPad - Even after enabling High Efficiency option, image from camera is JPEG. I have tested in both Chrome and Safari form iPad
2. Android - Samsung :  HEIC option was possible only when sharing the file.  Camera setting do not have this feature. So images are always JPEG
3. Laptop - HEIC images from laptop using Chrome browser is not working. ImageReducer error and its disabled for now.  But from Safari browser, even if HEIC is uploaded, it converted to JPEG and uploaded successfully with Timestamp and Geolocation.
